### ISSUE TITLE
[Filter] Remove "Title"-Filter

### ISSUE THREAD

**Issue by marwin89** [bohning/usdb_syncer#186]
### Suggestion:

It would be cool if you remove "Title"-Filter in usdb-syncer.

### Why:

- This filter lists every title of the whole usdb. Which doesn't make sense as a dropdown-select field (with over 10.000 entries).
- Titles should be just searched in the searchfield.

### Estimation
- Filler task: low effort, low impact.

Thank you 🚀

**Comment by RumovZ** [comment]
I agree that it doesn't help with finding songs. However, after the last GUI rework (apart from ditching the dropdown) it allows you to filter for specific titles, which the free text search field doesn't.
Maybe you're still right, but since it doesn't cause any harm, I would first like to see how it pans out in its new form before removing it.

**Comment by marwin89** [comment]
That's fine for me, take your time and try things out. Thanks for making syncer possible 🙏
